meinberlin/apps/votes/migrations/0004_alter_votingtoken_token.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.17 on 2023-02-16 13:45
+
+from django.db import migrations, models
+import meinberlin.apps.votes.models
+
+
+def delete_all_voting_token(apps, schema_editor):
+    VotingToken = apps.get_model("meinberlin_votes", "VotingToken")
```

Comment:
this again is slow with a lot of tokens but hopefully we shouldn't have that many?

Reply:
Did you check if it is maybe faster to just drop the whole table (like in the migration before) and then recreate it?

Reply:
no, not yet, but dropping and recreating adds even more code :/ the migrations are already not very nice for the votes app. I'll give it a try / copy it from the previous

Reply:
The dropping turned out to be very easy, it's just the first bit with the delete from the migration before. And then I added the recreation to the same table, so that even if the old migration that was in main before would have been run in the meantime, it would all still be fine. here we can also just do two migrations:
1. create one with `--empty` and add the delete
2. just make new migrations recreating the tables with the new state of the model